src/shogun/features/RandomFourierDotFeatures_benchmark.cc

```diff
@@ -9,6 +9,7 @@
 #include "shogun/features/DotFeatures_benchmark.h"
 #include "shogun/features/RandomFourierDotFeatures.h"
 #include "shogun/mathematics/UniformIntDistribution.h"
+#include "shogun/mathematics/linalg/LinalgNamespace.h"
```

Comment:
Why did you add the header file here?

Reply:
I thought I'd add a unit test inside this benchmark, using LinalgNamespace, too early though.

Reply:
unit tests go to unit tests, benchmarks go to benchmarks